documentation/functions/domain/SOA.md

```diff
@@ -24,12 +24,12 @@ parameter_types:
 
 ```javascript
 D("example.com", REG_THIRDPARTY, DnsProvider("DNS_BIND"),
-  SOA("@", "ns3.example.org.", "hostmaster.example.org.", 3600, 600, 604800, 1440),
+  SOA("@", "ns3.example.org.", "hostmaster@example.org", 3600, 600, 604800, 1440),
 );
 ```
 
 ## Notes
-
+* Previously, the accepted format for the SOA mailbox field was `hostmaster.example.org`. This has been changed to `hostmaster@example.org`
```

Comment:
Suggestion:

* Previously, the accepted format for the SOA mailbox field was `hostmaster.example.org`. This has been changed to `hostmaster@example.org`. DNSControl will change the `@` to `.` when generating the SOA record.

Reply:
The latter sentence is only accurate for the bind provider. I'd leave that out.